### DISCUSSION TITLE
V1 | Planning the Settings Page for Social Media Blocks! 🚀🚀🚀

### DISCUSSION THREAD

**Discussion by JavascriptDon** [JavascriptDon/Social-Media-Blocks-Extension#9] [Ideas]
## Thank you @Tyru5  for laying the foundation for the settings view.  😊

On the topic of settings, I've decided to create a mockup using Figma for what I have planned for this extension. 

### Here's the first mockup: 

![Figma.png](https://user-images.githubusercontent.com/101202952/180778215-15692460-52fd-478c-b8da-64cc1c007e6e.png)

### What I'm addressing here  in this mockup: 

**Theme functionality** 

-  To give the user the ability to change the color for the extension between **dark** and **light** mode.

**Updating filters** 
 
- To give the user the ability to update the filter settings internally for more effective blocking. 

Please comment down below with what you want to see as part of the extension!  👇

**Comment by LucidSamuel** [comment]
Great work @JavascriptDon
